virttest/qemu_vm.py

```diff
@@ -186,6 +191,26 @@ def __init__(self, name, params, root_dir, address_cache, state=None):
         self.start_monotonic_time = 0.0
         self.last_boot_index = 0
         self.last_driver_index = 0
+        self._caps = Capabilities()
+
+    def _probe_capabilities(self):
+        """ Probe whether the vm sets the capabilities. """
+        ver_dict = utils_misc.get_qemu_version(self.params)
+        ver = '.'.join(
+            (str(ver_dict['major']), str(ver_dict['minor']), str(ver_dict['update'])))
```

Comment:
Suggest: `ver = '{major}.{minor}.{update}'.format(**ver_dict)`

Reply:
ok, will update it. thanks.